pwiz_tools/Skyline/Model/Results/RemoteApi/Ardia/ArdiaUrl.cs

```diff
@@ -248,20 +194,13 @@ public override MsDataFileImpl OpenMsDataFile(OpenMsDataFileParams openMsDataFil
             var response = client.GetAsync(presignedUrl, HttpCompletionOption.ResponseHeadersRead, openMsDataFileParams.CancellationToken).Result;
             response.EnsureSuccessStatusCode();
             var responseStream = response.Content.ReadAsStreamAsync().Result;
-            using (var progressStream = new ProgressStream(responseStream, RawSize ?? 1))
+            var progressStream = new ProgressStream(responseStream, RawSize ?? 1);
```

Comment:
Why did you remove automatic dispose?

Reply:
I wanted to make it clear that calling Dispose on ProgressStream does not do anything (specifically, ProgressStream never closes the Stream that was passed into its constructor).

I could certainly put the word "using" back.
I am not sure whether having the word "using" on that line makes the code more or less clear and correct.

What do you think?

Reply:
I think a general rule we could follow is if something has a Dispose() and it has a lifetime that should be limited to the current scope, we should `using` it. You can add a comment like `// does not close stream` ?

Reply:
My rationale for applying the rule is that somebody might modify ProgressStream to add some other Dispose()able member, and assume that all callers are already Dispose()ing, so not revisit all the call sites to add `using` back.